adapters/openx/openx.go

```diff
@@ -298,3 +306,51 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 	}
 	return bidder, nil
 }
+
+func getBidMeta(bid *openrtb2.Bid) *openrtb_ext.ExtBidPrebidMeta {
+	if bid.Ext == nil {
+		return nil
+	}
+
+	var ext *oxBidExt
+	if err := jsonutil.Unmarshal(bid.Ext, &ext); err != nil {
+		return nil
+	}
+
+	buyerId := getBuyerIdFromExt(ext)
+	dspId := getDspIdFromExt(ext)
+	brandId := getBrandIdFromExt(ext)
+	if buyerId <= 0 && dspId <= 0 && brandId <= 0 {
+		return nil
+	}
+
+	return &openrtb_ext.ExtBidPrebidMeta{
+		NetworkID:    dspId,
+		AdvertiserID: buyerId,
+		BrandID:      brandId,
+	}
+}
+
+func getBuyerIdFromExt(ext *oxBidExt) int {
+	if ext.BuyerId == nil {
+		return 0
+	}
+	buyerID, _ := strconv.Atoi(*ext.BuyerId)
```

Comment:
Let's handle error which might be returned from `strconv.Atoi`
```suggestion
	buyerID, err := strconv.Atoi(*ext.BuyerId)
	if err != nil {
	    return 0
    }
```
and also add test cases for this scenario.

Reply:
Added, tanks for suggestion